tools/adam/adam_transform_GATK.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+#SET Argument variables
+#HDFS_INPUT_PATH_FILE=/user/vijaym/adam/NA12878.bam
+#HDFS_OUTPUT_PATH_FILE=/user/vijaym/adam/NA12878_CCC_TEST.adam
+HDFS_INPUT_PATH_FILE=/user/vijaym/adam//NA12878.bam
+
+HDFS_ADAM_PATH_FILE=/user/vijaym/adam//NA12879.adam
+HDFS_VCF_PATH_FILE=/user/vijaym/adam//dbsnp_132.vcf
+HDFS_VAR_PATH_FILE=/user/vijaym/adam//dbsnp_133.var.adam
+HDFS_MKDUP_PATH_FILE=/user/vijaym/adam//NA12879.mkdup.adam
+HDFS_SORT_PATH_FILE=/user/vijaym/adam//NA12879.sort.adam
+HDFS_RI_PATH_FILE=/user/vijaym/adam//NA12879.ri.adam
+HDFS_BQSR_PATH_FILE=/user/vijaym/adam//NA12879.bqsr.adam
```

Comment:
No hard-coded paths to input/output datasets should be within the Galaxy git repo.